STM/vertx/README.md

```diff
@@ -0,0 +1,181 @@
+OVERVIEW
+--------
+
+Another example of STM and Vert.x integration. The quickstart is divided into 4 parts each introducing
+a different facet of STM and Vert.x integration.
+
+Demo 1 shows a single JVM running multiple vertx instances sharing the same volatile STM object.
```

Comment:
There seems to be some inconsistency in naming. Sometimes it's "Demo 1", sometimes it's "Use Case 1"